components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js

```diff
@@ -17,7 +17,7 @@
  */
 define(['require', 'log', 'jquery', 'backbone', 'lodash', 'designViewUtils', 'dropElements', 'dagre', 'edge',
         'windowFilterProjectionQueryInput', 'joinQueryInput', 'patternOrSequenceQueryInput', 'queryOutput',
-        'partitionWith', 'jsonValidator'],
+        'partitionWith', 'jsonValidator', 'bootstrap'],
```

Comment:
I think we do not need this import "bootstrap" here .

Reply:
Removed the Bootstrap definition from  design-grid.js and drop-element.js

Reply:
![sp1](https://user-images.githubusercontent.com/29033481/49129796-2831db00-f2f7-11e8-848c-fcc32d9ea1c6.png)
![sp2](https://user-images.githubusercontent.com/29033481/49129797-2831db00-f2f7-11e8-803d-f99c3e8831dc.png)